extension/src/stabilization_info.rs

```diff
@@ -14,6 +14,9 @@ crate::functions_stabilized_at! {
     "1.16.0" => {
         approx_count_distinct(anyelement),
         approx_count_distinct_trans(internal,anyelement),
+        accessorpercentilearray_in(cstring),
+        accessorpercentilearray_out(accessorpercentilearray),
+        arrow_uddsketch_approx_percentile_array(uddsketch,accessorpercentilearray),
```

Comment:
This last function shouldn't go here, the entry you added for `STABLE_OPERATORS` should cover it.  Do you get an error without this entry?

Reply:
It passes without that entry, I will delete it.

Reply:
If I delete that line, the test fails with:
```
unexpectedly released features: [
    "function arrow_uddsketch_approx_percentile_array(uddsketch,accessorpercentilearray)",
]
```